notebooks/official/prediction/sdk_private_endpoint.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<p>Title is missing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2354/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>